src/main/java/com/example/finalproject/model/Category.java

```diff
@@ -0,0 +1,39 @@
+package com.example.finalproject.model;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+public class Category {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Enumerated(EnumType.STRING)
```

Comment:
Dalejmy enter przed adnotacjami - to się tyczy wszystkich adnotacji w projekcie

Reply:
poprawione